ATT07701.eslintrc

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Is your editor seeing this file or it should be renamed to ".eslintrc"?

Reply:
Fixed

Reply:
In the first place my editor did not seing it. After renaming to .eslintrc - it was available, therefore, I like more the identation of mine editor.